client/app.jsx

```diff
@@ -48,9 +49,11 @@ export default class App extends React.Component {
   }
 
   render() {
+    const token = window.localStorage.getItem('react-context-jwt');
+    const decoded = jwtDecode(token);
```

Comment:
We don't grab stuff like this in `render()` because the app can render over and over, and re-running these operations is really inefficient. instead, just do it one time in componentDidMount and store these in App's state